src/config/index.js

```diff
@@ -9,6 +9,7 @@ export const config = () => ({
   gp2gpMessengerAuthKeys: process.env.GP2GP_MESSENGER_AUTHORIZATION_KEYS,
   ehrRepoServiceUrl: process.env.EHR_REPO_SERVICE_URL,
   ehrRepoAuthKeys: process.env.EHR_REPO_AUTHORIZATION_KEYS,
+  rateLimitTimeoutMilliseconds: process.env.EHR_REPO_RATE_LIMIT_TIMEOUT_MILLISECONDS || 0,
```

Comment:
```suggestion
  rateLimitTimeoutMilliseconds: process.env.FRAGMENT_TRANSFER_RATE_LIMIT_TIMEOUT_MILLISECONDS || 0,
```
I believe it's more suitable to called it "fragment transfer" rather than "ehr-repo", as the timeout is occuring within the transferring out process rather than within ehr-repo.